components/audio.js

```diff
@@ -0,0 +1,19 @@
+const { Component } = require("../component");
+
+const ID = "audio";
+
+class Audio extends Component {
+    constructor() {
+        super(ID);
+        this.addCommand("-play (.*)", this.playAudio);
+    }
+
+    playAudio(fileName) {
```

Comment:
Maybe want to be able to omit the file extension in the fileName when parsing, could be a later improvement

Reply:
Ill add it to the issue backlog